superset-frontend/plugins/plugin-chart-table/src/Styles.tsx

```diff
@@ -40,8 +40,9 @@ export default styled.div`
   }
   th svg {
     color: ${({ theme: { colors } }) => colors.grayscale.light2};
-    position: relative;
+    position: absolute;
     vertical-align: middle;
+    top: ${({ theme: { gridUnit } }) => gridUnit * 2}px;
```

Comment:
Should we add `right: 0`? You can see on the screenshot that the arrows overlap with the th text on the right
<img width="726" alt="image" src="https://user-images.githubusercontent.com/15073128/154997233-f4c60e13-43c2-465c-ba0e-0c232d576778.png">

Reply:
Nice catch. However, I could not reproduce it locally. That's weird. I pushed the change with your suggestion.